dotnet/src/Extensions/PromptTemplates.Handlebars/Extensions/HandlebarsKernelExtensions.cs

```diff
@@ -18,13 +19,15 @@ public static class HandlebarsKernelExtensions
     /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
     /// <param name="promptTemplate">Plain language definition of the prompt, using Handlebars prompt template language</param>
     /// <param name="arguments">The operation arguments</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
```

Comment:
For all of these, can you use the same comment that's used on the other token params?